app/Http/Controllers/CampaignsController.php

```diff
@@ -78,11 +79,13 @@ public function showCampaign($id)
     {
         $campaign = $this->campaignService->find($id);
         $totals = $this->campaignService->getPostTotals($campaign);
+        $export = Signup::whereNotNull('details')->where('campaign_id', $id)->get();
 
         return view('pages.campaign_single')
             ->with('state', [
                 'campaign' => $campaign,
                 'initial_posts' => 'accepted',
+                'export' => ! $export->isEmpty(),
```

Comment:
I wonder if we need more control here re: whether to show the export button or not. Since we might want to use the `details` column for things that won't be exported. So, for example, what if we only decide to show the `export` button if `truth-opt-out` is in the `details column for any of the signups. It's a little tied to the business logic, but we can contain that piece in the controller and extract it easily, if we need to.

Reply:
@sbsmith86 I definitely like the idea of that but I'm wondering if we should wait until a case like that come up to support that? Or are we pretty sure that that is going to happen?

Reply:
I feel like it can't hurt to show the export button regardless, especially if coupled with your suggestion above about visually de-emphasizing this & providing more context to the admin.

Reply:
I guess i was just concerned with us making the assumption that anytime something is in the `details` field, we want to export something. But we can leave it there if you want. Would definitely push for it being de-emphasized, then.